### DISCUSSION TITLE
Keyboard shortcut keybindings - show bound keys

### DISCUSSION THREAD

**Discussion by decuser** [linuxmint/discussions#107] [Ideas]
Currently, you have to know what function is bound to a key to change its keybinding. It can be very difficult to find what function is bound to a key press. It would be an improvement, if it were possible to view what keys are bound to what functions as a list of keybindings by key. As an example, Move Window to Workspace Above is bound to Shift-Ctl-Alt-Up. Right now the only way to find this out is to expand Windows, then Inter-workspace, then look at the function "Move window to workspace above" and see that it's one of the bindings. It'd be great if it would appear in a list of bindings as Shift-Ctl-Alt-Up -> Move window to workspace above.

**Comment by Secret-chest** [comment]
Can you explain more?

**Comment by decuser** [comment]
Sure. I would like to see a list of all of the bindings in a table/grid:

| Key  | Function |
|-------------------|-------------------|
| CTL-F1 | Open Window |
| CTL-Alt-T | Open Terminal |
| Super-E | Open Nemo |
| Etc. | |

The table should include both delivered and customized keybindings (as set in the Keyboard app). Even if the grid is non-interactive, it would still be very handy, but if it were interactive, that'd be swell, too.

**Reply by root2185** [reply]
Look in `Help`.
Or press Ctrl+F1.

**Comment by decuser** [comment]
Nice! Is it new, or did I just miss it?

**Reply by root2185** [reply]
Not sure I was searching for "Open in Terminal".
